config/identifier-registration.yaml

```diff
@@ -0,0 +1,44 @@
+---
+# OpenID Connect client registry.
```

Comment:
Is this intended to be committed, or is it an accident?

Reply:
It's accidental. I'll remove it.